pom.xml

```diff
@@ -45,7 +45,8 @@
   <properties>
     <maven.assembly.version>2.2.1</maven.assembly.version>
     <hbase.version>0.92.1</hbase.version>
-    <cassandra.version>0.7.0</cassandra.version>
+    <cassandra.version>1.2.9</cassandra.version>
```

Comment:
that helps a bunch, thanks for getting back to me.

I suspect separate bindings are the way to go, but I agree that still working with older versions can be something we worry about later.